### DISCUSSION TITLE
conver .exe to .py

### DISCUSSION THREAD

**Discussion by AHAXPOHOC** [brentvollebregt/auto-py-to-exe#423] [Q&A] [answered]
hi! have some trouble...
i lost main.py file due to someone's mistake..
but i have converted .exe folder))
can i convert .exe to .py? i don't use github or another version control with this project((

**Comment by brentvollebregt** [comment, marked as answer]
You could try using [extremecoders-re/pyinstxtractor](https://github.com/extremecoders-re/pyinstxtractor) - this extracts the contents of a PyInstaller generated executable file
